widgets/nuxeo-file.html

```diff
@@ -67,19 +91,37 @@
       </dom-if>
     </div>
 
-    <dom-repeat items="[[files]]" as="file">
+    <label class="error" hidden$="[[!invalid]]">[[errorMessage]]</label>
+
+    <dom-if if="[[!multiple]]">
+      <template>
+        <dom-if if="[[value]]">
```

Comment:
I think it would be cheaper to have a single `if` with a computed binding with two dependencies instead of two dedicated ifs, once per dependency, since nothing else is happening between the inner and the outer `if`
```suggestion
        <dom-if if="[[_computedMethod(multiple, value]]">
```